.circleci/config.yml

```diff
@@ -13,10 +13,7 @@ jobs:
       - checkout
       - restore_cache:
           name: Restore Global Yarn Package Cache
-          keys:
```

Comment:
I think you might need to rebase again? Or was this change deliberate?

Reply:
Reverting config changes. Had some problems with yarn cache on circle, but let's see if it is fixed!

Reply:
Yup, config should be reverted. Had some caching issues on circle.